app/spec/controllers/caseworker/entries_controller_spec.rb

```diff
@@ -27,5 +24,17 @@
         expect(unescaped_body).to include("Log in with your LAN ID")
       end
     end
+
+    context "when state is disabled" do
+      before do
+        stub_site_config_value("sandbox", "staff_portal_enabled", false)
+      end
+
+      it "should show redirect to the root page" do
+        agency_short_name = site_config["sandbox"].agency_short_name
+        get :index, params: { site_id: "sandbox" }
+        expect(response).not_to be_successful
```

Comment:
nit: A better expectation would be something like

```suggestion
        expect(response).to redirect_to(root_url)
```